tests/common/protocol/test_extensions_goal_state_from_extensions_config.py

```diff
@@ -100,3 +100,16 @@ def test_it_should_parse_is_vm_enabled_for_rsm_upgrades(self):
             agent_families = protocol.get_goal_state().extensions_goal_state.agent_families
             for family in agent_families:
                 self.assertFalse(family.is_vm_enabled_for_rsm_upgrades, "is_vm_enabled_for_rsm_upgrades should be False")
+
+    def test_it_should_parse_encoded_signature_plugin_property(self):
+        data_file = wire_protocol_data.DATA_FILE.copy()
+        # This extensions goal state has 1 extension without encodedSignature property (ExampleHandlerLinux) and 1
+        # extension with encodedSignature property (ExampleExtension)
+        data_file["ext_conf"] = "wire/ext_conf-plugin_encoded_signature.xml"
```

Comment:
should we just add the signature to wire/ext_conf.xml and hostgaplugin/vm_settings.json so that most test will get the values? the signature will be a standard property now

Reply:
Updated